docs/getting-started.md

```diff
@@ -22,29 +22,30 @@ make docker-build
 MySQL setup
 -----------
 
-After you have MySQL up and running, create new empty database (/schema) & user for
+After you have MySQL up and running, create a new empty database (/schema) & user for
 `a12n-server`.
 
 ```sh
-mysql> CREATE DATABASE db_name;
-mysql> CREATE USER 'username'@'localhost' IDENTIFIED BY 'password';
+mysql> CREATE DATABASE a12nserver;
+mysql> CREATE USER 'a12nserver' IDENTIFIED BY 'password';
 ```
 
-The next step is to insert the MySQL schemas that are shipping with the git
-repository. The easiest is to just run:
+While running the application, it is possible to run into privileges issues.
+You need to enable the GRANT statement to grant privileges and roles.
 
 ```sh
-cat mysql-schema/*.sql | mysql -u username -p -h hostname databasename
+mysql> GRANT SELECT, INSERT, UPDATE, DELETE ON dbname.* TO 'a12nserver';
 ```
 
-While running the application, it is possible to run into privileges issues.
-You need to enable the GRANT statement to grant privileges and roles.
+The next step is to insert the MySQL schemas that are shipping with the git
+repository. The easiest is to just run:
 
 ```sh
-mysql> GRANT SELECT, INSERT, UPDATE, DELETE ON dbname.* TO 'username'@'localhost';
+cat mysql-schema/*.sql | mysql -u a12nserver -p -h hostname databasename
 ```
 
-Once you have finalized the permissions that you want to set up for your new users, always be sure to reload all the privileges.
+Once you have finalized the permissions that you want to set up for your new users,
+always be sure to reload all the privileges.
 
 ```sh
 mysql> FLUSH PRIVILEGES;
```

Comment:
I'd also just make this part of the earlier steps. It's just 4 mysql commands so I don't think there's a need for a lot of the 'flavor'